disruptor.go

```diff
@@ -15,6 +15,7 @@ import (
 
 	"github.com/grafana/xk6-disruptor/pkg/api"
 	"github.com/grafana/xk6-disruptor/pkg/kubernetes"
+	"k8s.io/client-go/rest"
```

Comment:
I think we should not make the main disruptor depend on  the `k8s.io/client-go/rest` package. I would prefer moving all the logic related to creating the config (including the `getKubernetesConfigPath` function) to the disruptor's kubernetes package.